docs/source/commands.rst

```diff
@@ -108,6 +108,22 @@ is provided via the palm context. If you want to use ``run_in_docker`` in your
 own command, ensure you use the ``@click.pass_obj`` decorator for your command,
 then use ``environment.run_in_docker(command)``.
 
+Note that in multi-service applications, ``run_in_docker`` will run proxy your command
```

Comment:
future iteration, should we have it say things like that with maybe an `--plan` flag? that instead of running the command, it describes in detail what the command would do?